geomesa-ui/src/main/ui/src/app/map/map.js

```diff
@@ -1,39 +1,43 @@
 angular.module('geomesa.map', [])
 
-    .directive('geomesaMap', [function () {
+    .directive('geomesaMap', ['$http', function ($http) {
+
         return {
             restrict: 'E',
             scope: {
                 map: '=?',
                 api: '=',
+                cards: '=',
```

Comment:
I think that instead of calling this cards we should just call it selectedPoints or something like that to keep it general. Most people won't know what a map "card" is.